cristin-organization/src/test/java/no/unit/nva/cristin/organization/query/QueryCristinOrganizationHandlerTest.java

```diff
@@ -225,7 +225,7 @@ void shouldReturnCorrectPayloadWhenRequestingVersion20230526() throws Exception
         assertThat(readHitsAsTree(actualHits), equalTo(readHitsAsTree(expectedHits)));
     }
 
-    private List<Organization> convertHitsToProperFormat(SearchResponse searchResponse) {
+    private List<Organization> convertHitsToProperFormat(SearchResponse<?> searchResponse) {
```

Comment:
nice, i assume this gets rid of a warning without suppressing it?

Reply:
yes